clab/netlink.go

```diff
@@ -150,28 +155,16 @@ func (c *cLab) createvethToBridge(l *Link) error {
 	return nil
 }
 
-// DeleteVirtualWiring deletes the virtual wiring
+// DeleteVirtualWiring deletes the virtual wiring by deleting network namespaces
 func (c *cLab) DeleteVirtualWiring(id int, link *Link) (err error) {
```

Comment:
I think we can get rid of the `id int` in this method signature, it's not used anywhere.

Reply:
in case one of the endpoints is a bridge, shouldn't we delete the interface that was added under that bridge? since containerlab created it during deploy

Reply:
removed `id` in 3f29ac3

for deletion of the interfaces of a bridge I need to check additionally, likely in a separate PR, as this functionality is not currently there

Reply:
Lgtm